src/app/factories/captcha.factory.ts

```diff
@@ -0,0 +1,41 @@
+import { celebrate, Joi } from 'celebrate'
+import { RequestHandler } from 'express'
+
+import FeatureManager, {
+  FeatureNames,
+  RegisteredFeature,
+} from '../../config/feature-manager'
+import { captchaCheck } from '../controllers/submissions.server.controller'
+
+// TODO(#144): Migrate middlewares and request handlers to the controller layer
+interface ICaptchaFactory {
```

Comment:
Can we move this to the `/types` folder

Reply:
Same reasoning I gave in the SmsFactory refactor; I use `typeof` to get the types leh, I scared circular dependency :X

Reply:
alright